captain/routes/test_sequence.py

```diff
@@ -34,13 +37,13 @@ async def websocket_endpoint(websocket: WebSocket, socket_id: str):
         logger.info(f"Client {socket_id} is disconnected")
 
 
-class DiscoverPytestParams(BaseModel):
+class DiscoverParams(BaseModel):
     path: str
     one_file: bool = Field(..., alias="oneFile")
 
 
 @router.get("/discover-pytest/")
```

Comment:
I think we shall rename the route to something like
`/discover/pytest`
`/discover/robot`

small detail haha